UnityProject/Assets/Scripts/Items/Wieldable.cs

```diff
@@ -0,0 +1,180 @@
+using Items;
+using Messages.Server;
+using Mirror;
+using UnityEngine;
+
+namespace Items
+{
+	public class Wieldable : NetworkBehaviour, IServerInventoryMove, ICheckedInteractable<HandActivate>
+	{
+		[SerializeField]
+		private int damageUnwielded;
+
+		[SerializeField]
+		private int damageWielded;
+
+		[SerializeField]
+		public ItemsSprites Wielded = new ItemsSprites();
+		public ItemsSprites Unwielded = new ItemsSprites();
```

Comment:
Public fields don't need the SerializedField attribute, that said prefer to use private fields and add the attribute. If the field is needed elsewhere, make a property so its intention is more explicit